tools/wire-format-reference.py

```
@@ -247,8 +276,25 @@ def _bin_twin(base: dict) -> dict:
 
 def generate() -> int:
     fixture = _load()
-    legacy, _ = _split_vectors(fixture)
-    fixture["vectors"] = legacy + [_bin_twin(v) for v in legacy]
+    legacy, bins = _split_vectors(fixture)
```

Ruff (0.16.2)

[warning] Unpacked variable `bins` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)